docs/executor/celery.rst

```diff
@@ -143,3 +143,38 @@ The components communicate with each other in many places
 * [9] **Scheduler** --> **DAG files** - Reveal the DAG structure and execute the tasks
 * [10] **Scheduler** --> **Celery's result backend** - Gets information about the status of completed tasks
 * [11] **Scheduler** --> **Celery's broker** - Put the commands to be executed
+
+Task execution process
+----------------------
+
+.. figure:: ../img/run_task_on_celery_executor.png
+    :scale: 50 %
+
+    Sequence diagram - task execution process
+
+Initially, two processes are running:
+
+- SchedulerProcess - process the tasks and run using CeleryExecutor
+- WorkerProcess - observes the queue waiting for new tasks to appear
+- WorkerChildProcess - waits for new tasks
+
+Two databases are also available:
+
+- QueueBroker
+- ResultBackend
```

Comment:
Can you describe shortly those dbs? Especially ResultBackend, is it the same database as Airflow db?

Reply:
They are described in the section above.

Reply:
I also do not describe what a worker and scheduler is, because I assume that a person is familiar with the previous section.

Reply:
These are different databases, but in popular configurations, they point to the same database server.